requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+# Linters
+pylama>=7.1.0
+pylama-pylint>=2.2.1
+isort>=4.2.5
```

Comment:
These are the linters we use in "python setup.py lint". Running this command is encouraged in docs to keep good documentation, style and avoid bugs.